src/SIM/SIMinput.C

```diff
@@ -599,13 +599,20 @@ bool SIMinput::parseBCTag (const tinyxml2::XMLElement* elem)
     if (code == 0) utl::getAttribute(elem,"code",code);
 
     IFEM::cout <<"\tRobin code "<< code;
-    if (!type.empty())
-      IFEM::cout <<" ("<< type <<")";
-    this->setPropertyType(code,Property::ROBIN);
-    if (rval)
-      this->setNeumann(rval->Value(),type,direction,code);
-    else
-      this->setNeumann(std::string(),type,direction,code);
+    if (type == "anasol")
+    {
+      IFEM::cout <<" (analytic)";
+      this->setPropertyType(code,Property::ROBIN_ANASOL);
+    }
+    else {
+      if (!type.empty())
+        IFEM::cout <<" ("<< type <<")";
+      this->setPropertyType(code,Property::ROBIN);
+      if (rval)
+        this->setNeumann(rval->Value(),type,direction,code);
+      else
+        this->setNeumann(std::string(),type,direction,code);
```

Comment:
or `this->setNeumann(rval ? rval->Value() : "", type, direction,code);`

Reply:
sure. not new code but we can improve while at it.